File: 1015/main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let lin1 = lines.shift().split(' ');
let lin2 = lines.shift().split(' ');

let x1 = lin1.shift();
let y1 = lin1.shift();
let x2 = lin2.shift();
let y2 = lin2.shift();

let dist = Math.sqrt(Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2));

console.log(dist.toFixed(4));

File: 1015/main.java
import java.io.IOException;
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        double x1,y1,x2,y2,res;
        
        Scanner input = new Scanner(System.in);
        
        x1 = input.nextDouble();
        y1 = input.nextDouble();
        x2 = input.nextDouble();
        y2 = input.nextDouble();
        
        res = Math.sqrt(Math.pow((x2-x1), 2) + Math.pow((y2-y1), 2));
 
        System.out.println(String.format("%.4f", res));
        
    }
 
}